en/lessons/libraries/guardian.md

```diff
@@ -130,17 +129,54 @@ The general flow of Guardian plug is:
 
 To meet all the needs of application developers, Guardian implements these phases separately. To find the token use the `Verify*` plugs.
 
+Guardian packages all of these up into a `Pipeline` to help you keep the logic together and test them independently.
 Let's create some pipelines.
 
 ```elixir
+defmodule MyApp.Guardian.StandardPipeline do
+  use Guardian.Plug.Pipeline, otp_app: :my_app,
+                              module: MyApp.Guardian,
+                              error_handler: MyApp.Guardian.ErrorHandler
+
+  plug Guardian.Plug.VerifySession, claims: %{"typ" => "access"}
+  plug Guardian.Plug.VerifyHeader, claims: %{"typ" => "access"}
+  plug Guardian.Plug.EnsureAuthenticated
+  plug Guardian.Plug.LoadResource, ensure: true
+end
+```
+
+This pipeline will
+
+* look for a token first in the session, if found it will verify it before moving on
+* If one is not found in the session it will look in the header and verify it if found. (In the Authorization header with value of "Bearer jwt")
+* Ensure authenticated will make sure that a token was found _somewhere_
```

Comment:
Shouldn't `Ensure authenticated` be [`EnsureAuthenticated`](https://github.com/elixirschool/elixirschool/pull/1158/files#diff-905f78632ecd759996eb649fac036c65R143) here?